presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -254,6 +260,30 @@ public void createPartitionedTableAs(HiveStorageFormat storageFormat)
         assertFalse(queryRunner.tableExists(getSession(), "test_create_partitioned_table_as"));
     }
 
+    public void createPartitionedTableAsWithIntegerColumns(HiveStorageFormat storageFormat)
+            throws Exception
+    {
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE test_create_partitioned_table_as " +
```

Comment:
this is the same table name used in the previous test, you need to change it to something else or this will fail when they run in parallel